### ISSUE TITLE
RDNA 3 support

### ISSUE THREAD

**Issue by radudiaconu0** [ROCm/ROCm#2098]
We have RDNA support or no in 5.5 release?
I ask this because THERE IS NOTHING IN DOCS. How can you have no windows support adn no rx 7900xtx supoprt in this release?

**Comment by mszpulak** [comment]
https://www.reddit.com/r/Amd/comments/135f8we/amd_rocm_55_in_the_process_of_being_released/?utm_source=share&utm_medium=android_app&utm_name=androidcss&utm_term=1&utm_content=share_button

**Comment by xfalcox** [comment]
This will be solved https://github.com/RadeonOpenCompute/ROCm/pull/2099 that is slated to the 5.6.0 milestone.

**Comment by NaMoCv** [comment]
> 

7900xt not work too..

**Comment by johnnynunez** [comment]
A lot is changing at RocM. Maybe in RocM 5.6(as they update GPU list) https://twitter.com/aschilling/status/1671447333635260416?s=20

**Comment by xfalcox** [comment]
Got my 7900XTX to work with PyTorch by installing nightly PyTorch for RoCM 5.5

**Comment by NaMoCv** [comment]
> A lot is changing at RocM. Maybe in RocM 5.6(as they update GPU list) https://twitter.com/aschilling/status/1671447333635260416?s=20

sry, i just forget reboot my machine

**Comment by TechScribe-Deaf** [comment]
Confirmed that it works on nightly preview for ROCm Pytorch for 7900 XTX on Arch Linux.

**Comment by radudiaconu0** [comment]
I tried to install rocm 5.5 on popos but i cant becxause it has a tooo new kernel

**Comment by johnnynunez** [comment]
anyone can execute: https://pypi.org/project/new-ai-benchmark/
```python
import tensorflow as tf
gpus = tf.config.experimental.list_physical_devices('GPU')
for gpu in gpus:
    tf.config.experimental.set_memory_growth(gpu, True)

from ai_benchmark import AIBenchmark
benchmark = AIBenchmark(verbose_level=2)
results = benchmark.run()
```

**Comment by briansp2020** [comment]
Are there still people who are waiting for 7900XTX support? Though the performance is still a bit poor, TensorFlow-upstream now runs when built on the latest ROCm release. I was looking into the status of ROCm support for 7900XTX and found a few issues opened by different people and wanted to link all to the issue I opened in MIOpen repo. Though there has not been any confirmation from the developer, I think the performance issues are due to insufficient optimization of MIOpen. 
https://github.com/ROCmSoftwarePlatform/MIOpen/issues/2342

**Comment by briansp2020** [comment]
@johnnynunez 
I ran the benchmark and posted the output to 
https://gist.github.com/briansp2020/e885f0eb6cbec45fcaf0c2eac8c3ee11

ROCm is in the arena trying stuff. Some will work, some won't. But always learning. j/k.

It actually looks better than I feard. I was really disappointed when I opened https://github.com/ROCmSoftwarePlatform/MIOpen/issues/2342 but it seems that was like one of the worst case scenarios. According to the benchmark output, there are many cases 7900XTX is optimized well enough. Still more work needs to be done. But I'm a lot more hopeful now.

BTW, does anyone know how to disable all the debug output from ROCm? I did os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3' trick to get rid of some. But ROCm still prints out too much debug output.

**Comment by johnnynunez** [comment]
>  Nice, There is room for improvement!

**Comment by johnnynunez** [comment]
> @johnnynunez I ran the benchmark and posted the output to https://gist.github.com/briansp2020/e885f0eb6cbec45fcaf0c2eac8c3ee11
> 
> ROCm is in the arena trying stuff. Some will work, some won't. But always learning. j/k.
> 
> It actually looks better than I feard. I was really disappointed when I opened [ROCmSoftwarePlatform/MIOpen#2342](https://github.com/ROCmSoftwarePlatform/MIOpen/issues/2342) but it seems that was like one of the worst case scenarios. According to the benchmark output, there are many cases 7900XTX is optimized well enough. Still more work needs to be done. But I'm a lot more hopeful now.
> 
> BTW, does anyone know how to disable all the debug output from ROCm? I did os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3' trick to get rid of some. But ROCm still prints out too much debug output.

Nice, There is room for improvement!

**Comment by ppanchad-amd** [comment]
@radudiaconu0 RDNA3 support is available as of latest ROCm 6.1.1. Thanks!
